hkSymposium/source/controller/MovieDetailViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  MovieDetailViewController.swift
+//  hkSymposium
+//
+//  Created by Anderson Santos Gusmão on 31/12/2017.
+//  Copyright © 2017 Anderson Santos Gusmão. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+class MovieDetailViewController: UIViewController {
+
+    @IBOutlet weak var posterImageView: UIImageView!
+    @IBOutlet weak var voteAverageProgressBar: UIProgressView!
+    @IBOutlet weak var labelTitle: UILabel!
+    @IBOutlet weak var labelReleaseDate: UILabel!
+    @IBOutlet weak var labelOverview: UILabel!
+    var movie: Movie!
```

Comment:
To prevent crashes you should make it optional (`var movie: Movie?`) and use `guard let..` when needed.